tools/scaffold/template/resource.tmpl

```diff
@@ -72,6 +72,10 @@ func (r *{{.NameCamelCase}}RS) Read(ctx context.Context, req resource.ReadReques
 
 	// connV2 := r.Client.AtlasV2
 	//if err != nil {
+	//	if apiResp != nil && apiResp.StatusCode == http.StatusNotFound {
```

Comment:
i'm not a fan of having comments about WHAT is being done as clear code can express that.
But sometimes comments with WHY can help.   I think in this case some comment like this could help:
// If resource is deleted outside of TF

Reply:
+1

Reply:
@maastha they explain what we are doing, but not why